node-swc/src/Visitor.ts

```diff
@@ -1184,8 +1184,8 @@ export default class Visitor {
   }
 
   visitObjectExpression(n: ObjectExpression): Expression {
-    if (n.properties) {
-      n.properties = this.visitObjectProperties(n.properties);
+    if (n.props) {
+      n.props = this.visitObjectProperties(n.props);
```

Comment:
This is wrong.
See https://github.com/swc-project/swc/blob/f5a90ae985c3a5dff80fb8f83facc5196093d4d7/ecmascript/ast/src/expr.rs#L184